multimodal/src/autogluon/multimodal/utils/onnx.py

```diff
@@ -7,99 +7,70 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    import tensorrt  # Unused but required by TensorrtExecutionProvider
+except:
+    logger.warning(
+        "Failed to import tensorrt package. "
+        "onnxruntime would fallback to CUDAExecutionProvider instead of using TensorrtExecutionProvider."
+    )
```

Comment:
Is it easy to install `tensorrt` for users? If not, consider making this a lazy import to avoid unnecessary warnings for users who don't need it.

Reply:
> Is it easy to install tensorrt for users?

Yes, it's just `pip install tensorrt`.

Reply:
tried lazy import, but didn't work.

Reply:
It's kind of weird why lazy import doesn't work. It's fine to keep it here for now. Maybe later dive into the reason.